components/WS/src/GeneralUtils.cpp

```diff
@@ -11,7 +11,6 @@
 #include <esp_system.h>
 #include <esp_wifi.h>
 #include <iomanip>
-#include <nvs.h>
 #include <sstream>
 #include <stdio.h>
 #include <string.h>
```

Comment:
Isn't it better to use '<cstdio>' or '<cstring>'?

Reply:
It's better to use the C++ style headers (<cstdio>), I plan to split this module and remove unused code from it. I plan to do it on another PR after this so others can start working on the code while I refactor that.

Reply:
Ok, this can be changed in another PR, before to complete the milestone